patterns/link-in-bio-wide-margins.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Title: Link in bio with profile, links and wide margins
+ * Slug: twentytwentyfive/link-in-bio-wide-margins
+ * Categories: banner, about, featured
+ * Description: A link in bio landing page with social links, a profile photo and a brief description.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:cover {"overlayColor":"secondary","isUserOverlayColor":true,"isDark":false,"align":"full","style":{"spacing":{"padding":{"right":"var:preset|spacing|80","left":"var:preset|spacing|80","top":"var:preset|spacing|80","bottom":"var:preset|spacing|80"}}},"layout":{"type":"constrained"}} -->
```

Comment:
Is there a specific reason to use a cover for this pattern? I'm wondering if we could use the group block instead of the cover, so that the site admin can use the section styles (this pattern would use the `Style 5`).

Reply:
Yes, great question @juanfra! It's because the cover is meant to be full-height, which I think I forgot to toggle on this pattern. But now that you mention it, it may be possible with group as well.

Reply:
Can you take a look now? I just updated it to use a group block with 100vh.

Reply:
It looks perfect! I believe the horizontal block spacing for the columns needs a bit of adjustment (to look like the original). 

I'm pushing all the changes for you to try it out ✨

Reply:
Thanks, this looks perfect with the changes!